### ISSUE TITLE
Invalid training loss

### ISSUE THREAD

**Issue by hshiah** [WuJunde/MedSegDiff#31]
The training loss on brats2020 of new version is usually nan.

<img width="151" alt="微信图片_20230211123834" src="https://user-images.githubusercontent.com/99376359/218240267-6e01067f-71eb-4c31-bd7d-0c6c16555941.png">

**Comment by hshiah** [comment]
When the loss is not NAN, the grad_norm is extremely large like 7.44e+04, while the previous version is usually around 10.
May I ask the reason? I train the model on raw brats2020 training data.

**Comment by WuJunde** [comment]
I fixed the bug, please update the project and try again.

**Comment by hshiah** [comment]
Hi, I tried the newest version and the model is stuck at training stage. I checked the GPU memory usage and it keeps a small value (around 2500 MiB) instead of normal value.
![image](https://user-images.githubusercontent.com/99376359/218323617-dd1eb2dc-9837-49d0-a78f-3e3fc9e8d0af.png)

**Comment by WuJunde** [comment]
@hshiah I checked it again, it works fine in my workplace. Did you run it on GPU? You may need to add --gpu 0.

**Comment by hshiah** [comment]
I tried it again and it works. Sorry for my mistake.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
